File: PrimeNumbers.java
public class PrimeNumbers {
    public static void printPrimesUnder1000() {
        System.out.println("Prime numbers under 1000:");
        for (int i = 2; i < 1000; i++) {
            if (isPrime(i)) {
                System.out.print(i + " ");
            }
        }
        System.out.println();
    }

    public static boolean isPrime(int n) {
        if (n <= 1) {
            return false;
        }
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args) {
        printPrimesUnder1000();
    }
}

File: PrimeNumbers.kt
class PrimeNumbers {
    fun getPrimesLessThan(n: Int): List<Int> {
        val primes = mutableListOf<Int>()
        for (i in 2 until n) {
            if (isPrime(i)) {
                primes.add(i)
            }
        }
        return primes
    }

    fun isPrime(n: Int): Boolean {
        if (n <= 1) return false
        var i = 2
        while (i * i <= n) {
            if (n % i == 0) return false
            i++
        }
        return true
    }
}

fun main() {
    val primeNumbers = PrimeNumbers()
    val n = 50
    val primes = primeNumbers.getPrimesLessThan(n)
    println("Prime numbers less than $n:")
    println(primes)
}
